tests/test_nbadapter.py

```diff
@@ -123,6 +123,29 @@ def test_nbadapter_repo(test_notebook_repo):
 
         validate_oda_dispatcher(nba)
 
+@pytest.mark.skip(reason="Not implemented")
+def test_nbadapter_lfs_repo(test_notebook_lfs_repo):
+    from nb2workflow.nbadapter import NotebookAdapter, find_notebooks, validate_oda_dispatcher
+
+    nbas = find_notebooks(test_notebook_lfs_repo)
+
+    assert len(nbas) >= 1
+
+    for nba_name, nba in nbas.items():
+        print("notebook", nba_name)
+
+        if os.path.exists(nba.output_notebook_fn):
+            os.remove(nba.output_notebook_fn)
+
+        if os.path.exists(nba.preproc_notebook_fn):
+            os.remove(nba.preproc_notebook_fn)
+
+        try:
+            nba.execute(dict())
+            assert False, 'nba.execute is expected to fail'
+        except Exception as ex:
+            assert ex.message == "git-lfs is not initialized"
```

Comment:
wait, so is the test working, i.e. the failure successfully reproduced?

Reply:
Unfortunately no. As you remember when we tried to reproduce the error we did not succeed unless we mount the existing git repository via -v option in docker provided that git versions inside docker and host machine are different. Even this effect depends on the particular version pairs. For this reason we decided to mark the test code as disabled.